magento/graphql-ce
# Added HTML renderer for description and short description

### Description
This PR improves output for description and shord_description of a product. Previously we just had a raw output. Now, if you use some directive (upon inserting a block into the description or so), its contents are rendered.

### Fixed Issues (if relevant)
1. https://github.com/magento/graphql-ce/issues/134: GraphQL API returns rendered content for product

### Manual testing scenarios
1. Magento installation from `2.3-develop` branch (optionally: with sample data installed)
2. Add a static block to a product description or short description. Example (for sample data)

```
A plain text description is here ...
...
{{block id="sale-block"}}
```
3. Get the product's description and the short description using a GraphQL request.
4. You should see rendered values in the place where the block is inserted
